src/checkout/components/ItemsRecap/ItemsRecap.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Grid } from "../../../_shared";
+import {
+  calculateTotalFromItems,
+  useSelectProductsFromCart,
+} from "../../../cart";
+import { OrderDetailRecap } from "../OrderDetailRecap";
+import { OrderTotalRecap } from "../OrderTotalRecap";
+
+export const ItemsRecap = () => {
+  const items = useSelectProductsFromCart();
```

Comment:
useSelect discloses implementation details that are not necessary, i would rather use something like "useCartProducts" or "useProducts in cart", or just useCart, returning items as an object